class.photon.php

```diff
@@ -305,8 +322,8 @@ public static function filter_the_content( $content ) {
 						// Remove the width and height arguments from the tag to prevent distortion
 						$new_tag = preg_replace( '#(?<=\s)(width|height)=["|\']?[\d%]+["|\']?\s?#i', '', $new_tag );
 
-						// Tag an image for dimension checking
-						$new_tag = preg_replace( '#(\s?/)?>(\s*</a>)?$#i', ' data-recalc-dims="1"\1>\2', $new_tag );
+						// Add srcset tag and tag an image for dimension checking
+						$new_tag = preg_replace( '#(\s?/)?>(\s*</a>)?$#i', ' srcset='.$srcset.' data-recalc-dims="1"\1>\2', $new_tag );
```

Comment:
Style-wise, we normally prefer to add spaces between strings and the append `.` character and variables, so it should look more like:

```php
'foo' . $bar . 'baz';
```